packages/web/src/css/style.css

```diff
@@ -0,0 +1,106 @@
+body {
+  background-color: #313744;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  min-height: 100vh;
+  max-width: 100vw;
+  overflow-x: hidden;
+}
+#Fullstacks-Logo {
+  margin-top: 48px;
+  max-width: 100%;
+}
+#Date-Text {
+  font-size: 2em;
+  font-family: "Rajdhani-Bold";
+  font-weight: 700;
+  color: white;
+  margin-bottom: 20px;
+}
```

Comment:
CSS is not case-sensitive, therefore try write anything in lowercase to prevent confusion.